flump/view.py

```diff
@@ -153,6 +154,13 @@ def _build_entity_data(self, entity):
                           entity, EntityMetaData(self._get_etag(entity)))
 
 
+def add_content_type(response):
```

Comment:
In this case I think it's safe to just do it blanket: if we've got a flump view then we know it's jsonapi...